update timings

K-Meech opened the pull request:
Closes https://github.com/HealthBioscienceIDEAS/microscopy-novice/issues/28

Update timings on all episodes based on initial teaching. @davecash75 - feel free to modify these, they are very rough estimates!

davecash75 reviewed (commented):
So if we do 10-5 minus hour for lunch and thirty minutes for breaks, that leaves 5.5 hours per day, or 11 hours total (660 minutes). We can subtract another 60 minutes as we ended at 4:30 both days. All told I think the current estimates add up to 495 minutes: 230+Housekeeping for day 1 . 265 for day 2 I think. So if anything, I think we probably think about upping the estimates for a couple (I think the image software and What is an Image may have been a touch longer), just so people feel like they don't have to rush them , and I would probably add it on to the exercises for most, as i think there will be more time needed when we get to a larger group.

K-Meech commented:
Thanks @davecash75 - I've updated the timing estimates. As we have 5 hours per day (if we finish at 4:30), this is about one hour per episode. I've updated the estimates so all episodes are estimated at 55 or 60 minutes to match this, adding time mostly to the exercises.

davecash75 commented:
Sounds good to me!
